Missing default configs

When following the binaries setup tutorial I ran into issued downloading the sql and ui files.

```
@cds:~$ /home/cds/cds/cds-engine download sql --config /home/cds/cds/conf.toml
Reading configuration file @ /home/cds/cds/conf.toml
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x21c21ee]

goroutine 1 [running]:
main.glob..func8(0x461d960, 0xc000280380, 0x0, 0x2)
        /tmp/QnVpbGQgYW5kIFBhY2thZ2UgQWxs/run/engine/cmd_download.go:95 +0xde
github.com/spf13/cobra.(*Command).execute(0x461d960, 0xc000280360, 0x2, 0x2, 0x461d960, 0xc000280360)
        /go/pkg/mod/github.com/spf13/cobra@v0.0.0-20171231101556-b95ab734e27d/command.go:750 +0x29d
github.com/spf13/cobra.(*Command).ExecuteC(0x461dde0, 0x0, 0x6, 0xc000098058)
        /go/pkg/mod/github.com/spf13/cobra@v0.0.0-20171231101556-b95ab734e27d/command.go:831 +0x2c9
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v0.0.0-20171231101556-b95ab734e27d/command.go:784
main.main()
        /tmp/QnVpbGQgYW5kIFBhY2thZ2UgQWxs/run/engine/main.go:26 +0x2d
```

This was due to these missing confs: 
```
DatabaseMigrate.Directory
UI.Staticdir
```
The doc doesn't mention them, I had to trace the code to find those

--- a/engine/cmd_download.go
+++ b/engine/cmd_download.go
@@ -78,6 +78,9 @@ var downloadUICmd = &cobra.Command{
 		if conf.API == nil {
 			sdk.Exit("Invalid configuration file")
 		}
+		if conf.UI == nil {
+			sdk.Exit("Invalid configuration file - missing ui section")
+		}
 		downloadTarGzFromGithub(conf.UI.Staticdir, "ui.tar.gz")
 	},
 }
@@ -91,6 +94,9 @@ var downloadSQLCmd = &cobra.Command{
 		conf := configImport(nil, flagDownloadConfigFile, flagDownloadRemoteConfig, flagDownloadRemoteConfigKey, "", "", false)
 		if conf.API == nil {
 			sdk.Exit("Invalid configuration file")
+		}
+		if conf.DatabaseMigrate == nil {
+			sdk.Exit("Invalid configuration file - missing databaseMigrate section")
 		}
 		downloadTarGzFromGithub(conf.DatabaseMigrate.Directory, "sql.tar.gz")
 	},

